Source/NSTableView.m

```diff
@@ -7028,4 +7138,14 @@ - (id) valueForKey: (NSString*)aKey
     }
 }
 
+- (NSMapTable *) _renderedViewPaths
```

Comment:
Providing full access to these instance variables looks wrong. Would it be possible to reduce this? Maybe ask for the view for a specific path and setting the view for a path?

Reply:
Agreed.  I think this is a cleaner approach.